spec/rubocop/cop/rspec/unspecified_exception_spec.rb

```diff
@@ -198,5 +198,29 @@
                                 ^^^^^^^^^^^^^^^ Specify the exception being captured
       RUBY
     end
+
+    it 'allows a subject function to be named raise_exception' do
+      expect_no_offenses(<<~RUBY)
+        def raise_exception
+          raise StandardError
+        end
+
+        expect {
+          raise_exception
+        }.to raise_error(StandardError)
+      RUBY
+    end
+
+    it 'allows a subject function to be named raise_error' do
+      expect_no_offenses(<<~RUBY)
+        def raise_error
+          raise StandardError
+        end
+
+        expect {
+          raise_error
+        }.to raise_exception(StandardError)
+      RUBY
+    end
```

Comment:
I just noticed that this spec file has a context named `'with raise_error matcher'`. So this example should probably be moved to around line 88 or so.

Reply:
Oh, also please squash the commits (as I imagine we are getting close to merging now).